packages/core/src/use-router.ts

```diff
@@ -0,0 +1,11 @@
+import {useRouter as useRouterByNext} from 'next/router'
```

Comment:
Maybe rename to `useNextRouter` or `useRouterNext`, `by` seems weird IMO

Reply:
Renamed to useRouterNext.